packages/common/test-utils.js

```diff
@@ -10,6 +10,15 @@ const fs = require('fs-extra');
 
 exports.inTestMode = () => process.env.NODE_ENV === 'test';
 
+/**
+ * Helper function to throw error for unit test exports
+ * @throws {Error}
+ */
+function throwTestError() {
```

Comment:
We talked a bit about this, but I'm thinking this might benefit from accepting many parameters as it will be used as a stand-in for some arbitrary function call.

Reply:
I *think* this does do that. 

```
->node
> function foo() {
... console.log(JSON.stringify(arguments));
... }
undefined
> foo('this', 'that')
{"0":"this","1":"that"}
undefined
> foo()
{}
undefined
> foo(['some', 'array'])
{"0":["some","array"]}
undefined
```

I could do something like function foo(...args) { //stuff } to explicitly use something other than arguments, but in this case that doesn't seem needed.